apps/e2e/cypress/support/featureFlagCommands.ts

```diff
@@ -0,0 +1,21 @@
+import * as dotenv from "dotenv";
+
+dotenv.config({ path: "../web/.env" });
+
+const overrideFeatureFlags = (flags: Object) => {
```

Comment:
Sorry, I didn't find time to review this before it was merged, but I'll leave this comment as a suggestion for next time...

@esizer My understanding is that we should basically never use `Object` in a typescript definition. I think a better typing for this would be `flags: Record<string, null | boolean>`.

Reply:
Oh yeah, good point thanks for flagging this! 🙇‍♀️ . I totally just had this while testing (wasn't sure exactly what types we could accept) and forgot to go back and adjust to a more strict type.

Reply:
Added an issue so this doesn't get buried and forgotten.

 - https://github.com/GCTC-NTGC/gc-digital-talent/issues/8780